.circleci/config.yml

```diff
@@ -196,29 +196,35 @@ workflows:
   workflow:
     jobs:
       - build-job:
-          name: "Xcode_11.4_iOS_13.4"
-          xcode: "11.4.0"
-          iOS: "13.4"
+          name: "Xcode_11.5_iOS_13.5"
+          xcode: "11.5.0"
+          iOS: "13.5"
+          device: "iPhone 11 Pro"
+      - build-job:
+          name: "Xcode_11.4.1_iOS_13.4.1"
+          xcode: "11.4.1"
+          iOS: "13.4.1"
           device: "iPhone 8 Plus"
       - build-job:
-          name: "Xcode_10.3_iOS_12.1"
-          xcode: "10.3.0"
-          iOS: "12.1"
+          name: "Xcode_11.4.1_iOS_12.2"
+          xcode: "11.4.1"
+          iOS: "12.2"
```

Comment:
Various test fixtures are conditionalized by iOS version, based on their file names. So we’ll need to rename or delete and rerecord a slew of test fixtures, similar to what took place in #2427.

Reply:
Thanks, fixtures were updated. I've also decided to remove iOS 13.4.1 build job since there is 13.5 already (this will allow to speed up build time and prevent adding additional fixtures for 13.4.1).